src/com/raghav/javabasics/FindMaxNumberInArray.java

```diff
@@ -0,0 +1,29 @@
+package com.raghav.javabasics;
+
+//Given an array of numbers, write a program to identify the max number in the list.
+
+public class FindMaxNumberInArray {
+    public static void main(String[] args) {
+        FindMaxNumberInArray result = new FindMaxNumberInArray();
+        result.findMax(new int[]{1,5,858,2,5,4,5,2158,1,587,4});
```

Comment:
What if the max number is duplicate in the array? Do you return it once or twice?